README.md

```diff
@@ -52,7 +52,14 @@ reproducible, saving you from sudden and unpleasant surprises.
 
 `ansible-core` Git revision. See https://github.com/ansible/ansible/tags
 and https://github.com/ansible/ansible/branches/all?query=stable- for
-ideas **(DEFAULT: `stable-2.13`)**
+ideas. The repository this refers to can be changed with the
+`ansible-core-repo` option. **(DEFAULT: `stable-2.13`)**
+
+
+### `ansible-core-repo`
+
+The repository from which to check out ansible-core
```

Comment:
I think this is an important clarification because there's a ton of other Git hostings out there.
```suggestion
The GitHub repository from which to check out `ansible-core`
```
Maybe this could be called an `ansible-core-github-repository-slug` or something like that? Just `ansible-core-repository-slug` maybe?

Reply:
This is something that should seldomly be used, so having something long is a good idea IMO. I'll switch to `ansible-core-github-repository-slug`.

Reply:
Implemented in 1860329ff87c812a596e33aed04394fdce20f6df.